cni-plugin/pkg/install/install.go

```diff
@@ -397,6 +397,7 @@ func writeCNIConfig(c config) {
 	name := getEnv("CNI_CONF_NAME", "10-calico.conflist")
 	path := winutils.GetHostPath(fmt.Sprintf("/host/etc/cni/net.d/%s", name))
 	err = os.WriteFile(path, []byte(netconf), perm)
+	os.Chmod(path, perm) // safeguarding since WriteFile does not change existing file's permissions
```

Comment:
This should be done after the `if err != nil` block below, and we should also check the error returned from `os.Chmod` in the same way.

Reply:
Done